src/app.ts

```diff
@@ -1,13 +1,14 @@
 import express from 'express';
 import 'express-async-errors';
 import cors from 'cors';
-import { json } from 'body-parser';
+import { json, urlencoded } from 'body-parser';
```

Comment:
Great work on this code. Just as a learning moment for myself. Why did you add urlencoded. I was able to remove it and still successful use the delete route. Is it for future use or what am I missing! Thanks!